vcpkg.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "nakama-sdk-cocos2d-x",
+    "version": "2.7.0",
+    "dependencies": [
+        {
+            "name": "nakama-sdk",
+            "version>=": "2.8.0#13",
+            "features": [
+                "apple-dylib"
```

Comment:
shouldn't it have `libcxx` feature?

Reply:
I accidentally forgot the push the new approach for Linux. I simplified Linux to this toolchain file: `cmake/Toolchain-Unreal-Linux.cmake`